packages/network/src/identifiers.ts

```diff
@@ -103,29 +90,25 @@ export type RtcConnectMessage = {
     }
 }
 
-export type LocalDescriptionMessage = {
-    subType: RtcSubTypes.LOCAL_DESCRIPTION
+export type RtcOfferMessage = {
+    subType: RtcSubTypes.RTC_OFFER
```

Comment:
Should this have `connectionId`?

Reply:
it should, strange why the compiler did not catch this